geomstats/geometry/general_linear.py

```diff
@@ -30,6 +35,33 @@ def inv(point):
         """Return the inverse of a matrix."""
         return gs.linalg.inv(point)
 
+    def random_uniform(self, n_samples=1, tol=1e-8):
```

Comment:
Numpy uses 1e-6 as a tolerance default, for example in np.isclose. You could use the same?